src/bulma.tsx

```diff
@@ -42,9 +42,7 @@ export declare namespace Bulma {
         isLoading?: boolean;
     }
 
-    export type Colors = 'white' | 'light' | 'dark' | 'black' | 'primary' | 'info' | 'success' | 'warning' | 'danger';
     export interface Color {
-        isColor?: Colors;
```

Comment:
Actually, this one should be:
```
export interface Color {
        isColor?: string;
    }
```

This interface is going to help us to do the destructing at some functions for `Bulma.Color`.